hack/run-e2e-tests.sh

```diff
@@ -25,11 +25,12 @@ then
         | go-junit-report -set-exit-code \
         > reports/kudo_e2e_test_report.xml
 
+    # Operators tests
     rm -rf operators
-    git clone https://github.com/kudobuilder/operators
+    git clone -b ad/test-with-manager-as-pod https://github.com/kudobuilder/operators
```

Comment:
Let's make sure that this branch is merged first and we can pull `master` again.